recipe/build-arrow.sh

```diff
@@ -89,6 +89,7 @@ cmake \
     -DARROW_WITH_ZLIB=ON \
     -DARROW_WITH_ZSTD=ON \
     -DCMAKE_BUILD_TYPE=release \
+    -DCMAKE_CXX_FLAGS_RELEASE="-O2" \
```

Comment:
I'd like to fit a comment in here somewhere, though it won't work in this list. We could set a variable above with an explanation and then use the variable here on this line perhaps.